.github/workflows/update_protos.yml

```diff
@@ -44,15 +44,35 @@ jobs:
         run: uv run make format
 
       - name: Add + Commit + Open PR
-        shell: bash
-        run: |
-          git checkout -b workflow/update-protos || git checkout workflow/update-protos
-          git add --all
-          git -c author.name=viambot -c author.email=viambot@users.noreply.github.com -c committer.name=GitHub -c committer.email=noreply@github.com commit -m "[WORKFLOW] Updating protos from ${{ github.event.client_payload.repo_name }}, commit: ${{ github.event.client_payload.sha }}"
-          git -c pull.ff=only pull origin workflow/update-protos || true  # do not fail if the branch doesn't exist
-          git push -f origin workflow/update-protos
-          gh pr view workflow/update-protos && gh pr reopen workflow/update-protos || gh pr create -B main -H workflow/update-protos -t "Automated Protos Update" -b "This is an auto-generated PR to update proto definitions. Check the commits to see which repos and commits are responsible for the changes" -a njooma -r njooma
+        id: cpr
+        uses: peter-evans/create-pull-request@v7
+        with:
+          commit-message: '[WORKFLOW] Updating protos from ${{ github.event.client_payload.repo_name }}, commit: ${{ github.event.client_payload.sha }}'
+          branch: 'workflow/update-protos'
+          delete-branch: true
+          title: Automated Protos Update
+          body: This is an auto-generated PR to update proto definitions. Check the commits to see which repos and commits are responsible for the changes
+          assignees: njooma
+          reviewers: njooma
+          token: ${{ secrets.GIT_ACCESS_TOKEN }}
```

Comment:
Switching to `peter-evans/create-pull-request`. This hasn't been tested in `python` specifically but we _are_ using this same workflow and version in the typescript SDK, which is passing (the historical failures in TS were related to the token we used and have been resolved since we switched to the `REPO_READ_TOKEN`).